scripts/dts/edtlib.py

```diff
@@ -1013,10 +1013,15 @@ def _binding_paths(bindings_dir):
     # Returns a list with the paths to all bindings (.yaml files) in
     # 'bindings_dir'
 
-    return [os.path.join(root, filename)
-            for root, _, filenames in os.walk(bindings_dir)
-                for filename in filenames
-                    if filename.endswith(".yaml")]
+    binding_files = []
```

Comment:
Could call it `binding_paths` maybe. Won't overlap with the function due to the `_`.